src/SignalR/common/Shared/SystemClock.cs

```diff
@@ -1,28 +1,14 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-
 namespace Microsoft.AspNetCore.Internal
 {
     /// <summary>
     /// Provides access to the normal system clock.
     /// </summary>
     internal class SystemClock : ISystemClock
     {
-        /// <summary>
-        /// Retrieves the current UTC system time.
-        /// </summary>
-        public DateTimeOffset UtcNow => DateTimeOffset.UtcNow;
```

Comment:
Does this mean there were no tests for `AuthenticationExpiration`?

Reply:
There are, but they aren't using the clock abstraction.